tests/tvos/view-with-example/tvos-view-with-example.test.js

```diff
@@ -39,12 +39,11 @@ jest.mock('prompts', () => args => {
 jest.mock('execa', () => (cmd, args, opts) => {
   mockCallSnapshot.push({ execa: [cmd, args, opts] })
   if (cmd === 'git') {
-    if (args[1] === 'user.email')
-      return Promise.resolve({ stdout: 'alice@example.com' })
-    else
+    if (args[1] === 'user.email') { return Promise.resolve({ stdout: 'alice@example.com' }) } else {
```

Comment:
For some reason `lint:fix` seems to miss some cases like this, at least on macOS and Linux.

Reply:
... but `lint:fix` on my Windows machine does not miss the cases like this.